[15:35:08] [DEBUG] Running client sanity checks...
[15:35:08] [DEBUG] No setup function found in play
[15:35:08] [DEBUG] Found extension class Music in play: Attempting to load
[15:35:08] [DEBUG] Added callback: Music._menu
[15:35:08] [DEBUG] Added callback: Music._pause
[15:35:08] [DEBUG] Added callback: Music._resume
[15:35:08] [DEBUG] Added callback: Music._skip
[15:35:08] [DEBUG] Added callback: Music._stop
[15:35:08] [DEBUG] Added callback: Music.play
[15:35:08] [DEBUG] Loaded Extension: play
[15:35:08] [DEBUG] No setup function found in db_refesh
[15:35:08] [DEBUG] Found extension class Database in db_refesh: Attempting to load
[15:35:08] [DEBUG] Added callback: Database.dbv1_command
[15:35:08] [DEBUG] Added callback: Database.dbv2_command
[15:35:08] [DEBUG] Loaded Extension: db_refesh
[15:35:08] [DEBUG] Added callback: __join
[15:35:08] [DEBUG] Added callback: __leave
[15:35:08] [DEBUG] Added callback: _about
[15:35:08] [DEBUG] Added callback: _help
[15:35:08] [DEBUG] Added callback: _refresh
[15:35:08] [DEBUG] Added callback: _setup
[15:35:08] [DEBUG] Added callback: _starup
[15:35:08] [DEBUG] 7 callbacks have been loaded from __main__.
[15:35:08] [DEBUG] Added callback: _disconnect
[15:35:08] [DEBUG] Added callback: _on_websocket_ready
[15:35:08] [DEBUG] Added callback: on_autocomplete_completion
[15:35:08] [DEBUG] Added callback: on_autocomplete_error
[15:35:08] [DEBUG] Added callback: on_command_completion
[15:35:08] [DEBUG] Added callback: on_command_error
[15:35:08] [DEBUG] Added callback: on_component_completion
[15:35:08] [DEBUG] Added callback: on_component_error
[15:35:08] [DEBUG] Added callback: on_error
[15:35:08] [DEBUG] Added callback: on_modal_completion
[15:35:08] [DEBUG] Added callback: on_modal_error
[15:35:08] [DEBUG] Added callback: on_resume
[15:35:08] [DEBUG] 12 callbacks have been loaded from Client.
[15:35:08] [DEBUG] Attempting to login
[15:35:09] [DEBUG] Caching ingested rate limit data for: 78bb8553d9352a5a2f89f9def401287a
[15:35:09] [DEBUG] GET /users/@me Has exhausted its ratelimit (1000)! Locking route for 0.001 seconds
[15:35:09] [DEBUG] GET /users/@me Received 200 :: [0/1000 calls remaining]
[15:35:09] [DEBUG] Caching ingested rate limit data for: d28a133af187e91d71d96a223467ce49
[15:35:09] [DEBUG] GET /oauth2/applications/@me Has exhausted its ratelimit (1000)! Locking route for 0.001 seconds
[15:35:09] [DEBUG] GET /oauth2/applications/@me Received 200 :: [0/1000 calls remaining]
[15:35:09] [DEBUG] Caching ingested rate limit data for: 0a6d458104b103c7be293cc9b284d9ee
[15:35:09] [DEBUG] GET /gateway Received 200 :: [99/100 calls remaining]
[15:35:09] [INFO] Shard ID 0 | Starting Shard
[15:35:09] [INFO] Shard ID 0 | Shard is attempting to connect to gateway...
[15:35:09] [DEBUG] Shard ID 0 | Identification payload sent to gateway, requesting intents: 3407613
[15:35:10] [INFO] Shard ID 0 | Gateway connection established
[15:35:10] [DEBUG] Shard ID 0 | Session ID: 9471bfa4904ba526137c29149716140c Trace: ['["gateway-prd-us-east1-d-2f71",{"micros":148540,"calls":["id_created",{"micros":1352,"calls":[]},"session_lookup_time",{"micros":349,"calls":[]},"session_lookup_finished",{"micros":12,"calls":[]},"discord-sessions-prd-2-78",{"micros":145863,"calls":["start_session",{"micros":98490,"calls":["discord-api-7949f7f68f-wn9d8",{"micros":87427,"calls":["get_user",{"micros":14400},"get_guilds",{"micros":9136},"send_scheduled_deletion_message",{"micros":17},"guild_join_requests",{"micros":422},"authorized_ip_coro",{"micros":11}]}]},"starting_guild_connect",{"micros":64,"calls":[]},"presence_started",{"micros":339,"calls":[]},"guilds_started",{"micros":182,"calls":[]},"guilds_connect",{"micros":2,"calls":[]},"presence_connect",{"micros":46756,"calls":[]},"connect_finished",{"micros":46762,"calls":[]},"build_ready",{"micros":23,"calls":[]},"clean_ready",{"micros":0,"calls":[]},"optimize_ready",{"micros":1,"calls":[]},"split_ready",{"micros":1,"calls":[]}]}]}]']
[15:35:10] [DEBUG] Dispatching Event: websocket_ready
[15:35:10] [DEBUG] Caching ingested rate limit data for: dd84d9818020c27975c0d5ab4a794d44
[15:35:10] [DEBUG] GET /applications/{application_id}/commands Has exhausted its ratelimit (1000)! Locking route for 0.001 seconds
[15:35:10] [DEBUG] GET /applications/{application_id}/commands Received 200 :: [0/1000 calls remaining]
[15:35:10] [DEBUG] Waiting for bucket dd84d9818020c27975c0d5ab4a794d44 to unlock.
[15:35:11] [DEBUG] Caching ingested rate limit data for: dd84d9818020c27975c0d5ab4a794d44
[15:35:11] [DEBUG] GET /applications/{application_id}/commands Has exhausted its ratelimit (1000)! Locking route for 0.001 seconds
[15:35:11] [DEBUG] GET /applications/{application_id}/commands Received 200 :: [0/1000 calls remaining]
[15:35:11] [DEBUG] 0 is already up-to-date with 19 commands.
[15:35:11] [DEBUG] Sync of 1 scopes took 0.7003814999998212 seconds
[15:35:11] [DEBUG] Dispatching Event: startup
[15:35:11] [DEBUG] Sending data to websocket: {"op": 3, "d": {"since": 1701506111017, "activities": [{"name": "l\u1ec7nh /help \u0111\u1ec3 gi\u00fap \u0111\u1ee1", "type": 0}], "status": "idle", "afk": false}}
[15:35:16] [DEBUG] Sending heartbeat every 41.25 seconds
[15:35:16] [DEBUG] Sending data to websocket: {"op": 1, "d": 6}
[15:35:16] [DEBUG] Shard ID 0 | ❤ Gateway is sending a Heartbeat
[15:35:17] [DEBUG] Shard ID 0 | ❤ Received heartbeat acknowledgement from gateway
[15:35:58] [DEBUG] Sending data to websocket: {"op": 1, "d": 6}
[15:35:58] [DEBUG] Shard ID 0 | ❤ Gateway is sending a Heartbeat
[15:35:58] [DEBUG] Shard ID 0 | ❤ Received heartbeat acknowledgement from gateway
[15:36:31] [DEBUG] 1174695146214928500::play should be called
[15:36:31] [DEBUG] Caching ingested rate limit data for: 432f6223265d156004ac81366c5a948f
[15:36:31] [DEBUG] POST /interactions/{interaction_id}/aW50ZXJhY3Rpb246MTE4MDQyNzI2MDk2MDk3MjgwMDp0MFVyZ2dlOG5MRjJRNmJvM0hrdWNyVkMzUHFZdHpZcnlxYzgyZW5CUWtpQm5OYkh1R0J1aEhXVWtJenNZUm44VHhXR1hzdzh1OURtVzlyWDBtN0pVVWFla2s3RXhMc0I1bVlMQTdXNENnNkNnSDlTendCUTlBRnNLM3NETU9BRQ/callback Received 204 :: [4/5 calls remaining]
[15:36:31] [DEBUG] Sending data to websocket: {"op": 4, "d": {"guild_id": 1176400128232402974, "channel_id": 1180418114001838110, "self_mute": false, "self_deaf": false}}
[15:36:31] [DEBUG] Waiting for voice connection data...
[15:36:31] [DEBUG] Dispatching Event: voice_user_join
[15:36:32] [DEBUG] Attempting to initialise voice gateway...
[15:36:32] [DEBUG] Voice Connection has identified itself to Voice Gateway
[15:36:32] [DEBUG] Discord send VC Ready! Establishing a socket connection...
[15:36:32] [DEBUG] IP Discovery in progress...
[15:36:32] [DEBUG] Voice Initial Response Received: b'\x00\x02\x00F\x00\x003y27.75.165.72\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\xf22'
[15:36:32] [DEBUG] IP Discovered: 27.75.165.72 #62002
[15:36:32] [DEBUG] Sending data to websocket: {"op": 1, "d": {"protocol": "udp", "data": {"address": "27.75.165.72", "port": 62002, "mode": "xsalsa20_poly1305_lite"}}}
[15:36:32] [DEBUG] Starting UDP Keep Alive
[15:36:32] [DEBUG] Unhandled OPCODE: 18 = data = {'user_id': '610841389164396565', 'flags': 2}
[15:36:32] [DEBUG] Unhandled OPCODE: 20 = data = {'user_id': '610841389164396565', 'platform': 0}
[15:36:33] [INFO] Voice connection established; using xsalsa20_poly1305_lite
[15:36:37] [DEBUG] Ending UDP Keep Alive
[15:36:39] [DEBUG] Sending data to websocket: {"op": 1, "d": 10}
[15:36:39] [DEBUG] Shard ID 0 | ❤ Gateway is sending a Heartbeat
[15:36:39] [DEBUG] Shard ID 0 | ❤ Received heartbeat acknowledgement from gateway
[15:36:53] [DEBUG] 1174695146214928500::play should be called
[15:36:54] [DEBUG] Caching ingested rate limit data for: 432f6223265d156004ac81366c5a948f
[15:36:54] [DEBUG] POST /interactions/{interaction_id}/aW50ZXJhY3Rpb246MTE4MDQyNzM1NjcwMDE1NTk5Njo5UXFEY3MzNGFRa2tEdk1qb2NES1p4bG82VGZDdHNZOHY3dE1zM08xTWpBRXNXSzh6TU5CNFFxa0dUcFdYZ3NGQjF4SURydFN3YlNCRjZrWGdpZ3FRbHVzaGxyY1JQQnMwcGFSZHp1TGRvVTQxNFVVRXZaOThWckF4dnBVSU5VdQ/callback Received 204 :: [4/5 calls remaining]
[15:36:54] [DEBUG] Sending data to websocket: {"op": 4, "d": {"guild_id": 1176400128232402974, "channel_id": 1180418114001838110, "self_mute": false, "self_deaf": false}}
[15:36:54] [DEBUG] Waiting for voice connection data...
[15:36:54] [DEBUG] Dispatching Event: voice_user_join
[15:36:59] [DEBUG] Dispatching Event: command_error
[15:36:59] [DEBUG] Dispatching Event: command_completion
[15:36:59] [DEBUG] Dispatching Event: error
[15:36:59] [INFO] Command Called: play with event.ctx.args = ['nụ cười 9'] | event.ctx.kwargs = {'song': 'nụ cười 9'}
[15:36:59] [ERROR] Ignoring exception in cmd `/play`:
Traceback (most recent call last):
  File "D:\Project\Python\qpneruy-Git\venv\Lib\site-packages\interactions\client\client.py", line 1900, in __dispatch_interaction
    response = await callback
               ^^^^^^^^^^^^^^
  File "D:\Project\Python\qpneruy-Git\venv\Lib\site-packages\interactions\client\client.py", line 1771, in _run_slash_command
    return await command(ctx, **ctx.kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Project\Python\qpneruy-Git\venv\Lib\site-packages\interactions\models\internal\command.py", line 132, in __call__
    await self.call_callback(self.callback, context)
  File "D:\Project\Python\qpneruy-Git\venv\Lib\site-packages\interactions\models\internal\application_commands.py", line 833, in call_callback
    return await self.call_with_binding(callback, ctx, *new_args, **new_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Project\Python\qpneruy-Git\venv\Lib\site-packages\interactions\models\internal\callback.py", line 43, in call_with_binding
    return await callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Project\Python\qpneruy-Git\music\play.py", line 170, in play
    await ctx.author.voice.channel.connect()
  File "D:\Project\Python\qpneruy-Git\venv\Lib\site-packages\interactions\models\discord\channel.py", line 2311, in connect
    return await self._client.connect_to_vc(self._guild_id, self.id, muted, deafened)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Project\Python\qpneruy-Git\venv\Lib\site-packages\interactions\client\client.py", line 2449, in connect_to_vc
    return await self._connection_state.voice_connect(guild_id, channel_id, muted, deafened)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Project\Python\qpneruy-Git\venv\Lib\site-packages\interactions\api\gateway\state.py", line 241, in voice_connect
    await voice_state.connect()
  File "D:\Project\Python\qpneruy-Git\venv\Lib\site-packages\interactions\models\internal\active_voice_state.py", line 158, in connect
    raise VoiceConnectionTimeout from None
interactions.client.errors.VoiceConnectionTimeout: Failed to connect to voice channel. Did not receive a response from Discord

[15:37:20] [DEBUG] Sending data to websocket: {"op": 1, "d": 13}
[15:37:20] [DEBUG] Shard ID 0 | ❤ Gateway is sending a Heartbeat
[15:37:20] [DEBUG] Shard ID 0 | ❤ Received heartbeat acknowledgement from gateway
[15:37:55] [DEBUG] Dispatching Event: disconnect
[15:37:55] [DEBUG] Stopping the bot.
[15:37:55] [INFO] Shard ID 0 | Stopping Shard
[15:37:55] [DEBUG] Dispatching Event: command_completion
[15:37:55] [INFO] Command Called: play with event.ctx.args = ['nụ cười 9'] | event.ctx.kwargs = {'song': 'nụ cười 9'}
